Geometry_Engine/Compute/InsertionVector.cs

```diff
@@ -0,0 +1,83 @@
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Base;
+using BH.oM.Geometry;
+using BH.oM.Base.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****          Public Methods                   ****/
+        /***************************************************/
+
+        [Description("Returns the insertion vector for element placement, dependent upon the number of elements and the start and end points.")]
+        [Input("elementCount", "Number of elements to place between the two points.")]
+        [Input("insertionStartPoint", "The start point of the insertion vector to create the Symbols.")]
+        [Input("insertionEndPoint", "The end point of the insertion vector to create the Symbols.")]
+        [Output("insertionVector", "The insertion vector with the correct length to create the Symbols.")]
+        public static Vector InsertionVector(Point insertionStartPoint, Point insertionEndPoint, int elementsCount = 1)
+        {
+            return (insertionStartPoint - insertionEndPoint) / (elementsCount == 1 ? 1 : elementsCount - 1);
+        }
+
+        /***************************************************/
+
+        [Description("Returns the orthogonal insertion vector as determined by two provided points, dependent upon which axis is more represented by the difference between the two points.")]
+        [Input("insertionStartPoint", "The start point of the insertion vector.")]
+        [Input("insertionEndPoint", "The end point of the insertion vector.")]
+        [Output("orthogonalInsertionVector", "The orthogonal insertion vector as determined by two provided points.")]
+        public static Vector OrthogonalInsertionVector(Point insertionStartPoint, Point insertionEndPoint, int elementsCount = 1)
```

Comment:
I think I get the concept after reading the code, but out of curiosity: what's the use of this in practice? Does it happen to you to distribute things based on clicks or similar? If so, it should be simpler to first make the specified vector orthogonal and then call `InsertionVector` - thanks to that, your distribution will be kept within a single line, while the newly added method querying the largest orthogonal term of the vector could be easily reused elsewhere.